website/src/main.rs

```diff
@@ -82,17 +84,19 @@ fn switch<G: Html>(route: ReadSignal<Routes>) -> View<G> {
             }
         },
         Routes::Docs(a, b) => {
-            let data = fetch_docs_data(format!("/static/docs/{a}/{b}.json"));
+            let data = fetch_docs_data(format!("/static/docs/{a}/{b}.json",));
+            let path: Signal<String> = create_signal(format!("{a}/{b}"));
```

Comment:
Type inference should work well enough here.
```suggestion
            let path = create_signal(format!("{a}/{b}"));
```